frontend/model/contracts/group.js

```diff
@@ -727,6 +734,20 @@ DefineContract({
         }
       }
     },
+    'gi.contracts/group/updateVotingRules': {
+      validate: objectOf({
+        ruleName: x => [RULE_PERCENTAGE, RULE_DISAGREEMENT].includes(x),
```

Comment:
I'm don't think `objectOf` supports this type of syntax...? So this should be `string`.

Reply:
I believe it does. It's the same we did for the method `grouPprofileUpdate`.

Reply:
That uses `objectMaybeOf`, which does support it